crates/services/relayer/src/ports/tests.rs

```diff
@@ -69,13 +69,13 @@ fn insert_always_raises_da_height_monotonically() {
     db.insert_messages(&5u64.into(), &messages[..5]).unwrap();
 }
 
-#[test_case(None, 0, 0)]
-#[test_case(None, 10, 10)]
-#[test_case(0, 10, 10)]
-#[test_case(0, None, 0)]
-#[test_case(10, 11, 11)]
-#[test_case(11, None, 11)]
-#[test_case(11, None, 10)]
+#[test_case(None, 0, 0; "can set DA height to 0 when there is none available")]
+#[test_case(None, 10, 10; "can set DA height to 10 when there is none available")]
+#[test_case(0, 10, 10; "can set DA height to 10 when it is 0")]
+#[test_case(0, None, 0; "inserts are bypassed when height goes from 0 to 0")]
+#[test_case(10, 11, 11; "can set DA height to 11 when it is 10")]
+#[test_case(11, None, 11; "inserts are bypassed when height goes from 11 to 11")]
+#[test_case(11, None, 10; "inserts are bypassed when height reverted from 11 to 10")]
```

Comment:
Hmm, that test case looks weird to me=) Maybe we need to return an error from `grow_monotonically` if the `new_height` is lower than the `current_height`.

My comment in the [initial PR](https://github.com/FuelLabs/fuel-core/pull/971#discussion_r1096189654) was left unrsponned=) Maybe you want to update it

Reply:
I agree. I didn't want to change the behavior of the code at all with this work. We should create another issues for fixing this IMO.

Reply:
Haha. I just saw the name of the tests lol: `set_raises_da_height_monotonically`. 

As I mentioned above, I think these tests need to all be rewritten anyway. There needs to be failure modes and checked output. These tests are kinda meaningless now. I'll put together an issue.

Reply:
https://github.com/FuelLabs/fuel-core/issues/1292